library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/PayloadType.kt

```diff
@@ -0,0 +1,8 @@
+package com.chuckerteam.chucker.internal.ui.transaction
+
+import java.io.Serializable
+
+sealed class PayloadType : Serializable {
```

Comment:
Two things here:

- Visibility should be `internal`
- This use case seems more appropriate for an `enum`, unless you had some future development in mind?

Reply:
Yeah, agree. I was in that mindset with other changes I have locally at the moment where some sealed classes have some data. 
Will update.